docs/subnets/customize-a-subnet.md

```diff
@@ -440,6 +441,83 @@ In addition to the AllowList interface, the FeeConfigManager adds the following
 
 Subnet-EVM contains example contracts for precompiles under `/contract-examples`. It's a hardhat project with tests, tasks. For more information see [contract examples README](https://github.com/ava-labs/subnet-evm/tree/master/contract-examples#subnet-evm-contracts).
 
+## Network Upgrades: Enable/Disable Precompiles
+
+:::warning
+
+Performing a network upgrade requires coordinating the upgrade network-wide. A network upgrade changes the rule set used to process and verify blocks, such that any node that upgrades incorrectly or fails to upgrade by the time that upgrade goes into effect may become out of sync with the rest of the network.
+
+:::
+
+In addition to specifying the configuration for each of the above precompiles in the genesis chain config, they can be individually enabled or disabled at a given timestamp as a network upgrade. Disabling a precompile disables calling the precompile and destructs its storage so it can be enabled at a later timestamp with a different configuration if desired.
+
+These upgrades can be specified in a file named `upgrade.json` placed in the same directory as [`config.json`](#chain-configs) using the following format:
```

Comment:
Do we print anything during startup that says which upgrades are configured and/or have an API to check? I imagine some people will put this in the wrong place.

Reply:
The chain config will be [printed](https://github.com/ava-labs/subnet-evm/pull/148/files#diff-10ff00a15ef58da7485fedeca3906c73cb236fde4b143a9d61601c63cbf1ffe8R144) during startup which includes the precompiles